aspnetcore/security/anti-request-forgery.md

```diff
@@ -221,14 +218,14 @@ In traditional HTML-based applications, antiforgery tokens are passed to the ser
 
 ### AngularJS
```

Comment:
Needs to call out the need to refresh the tokens after a login by redirecting to a view of some sort.